File: Functions2/ex09.c
#include <stdio.h>

void primes()
{
  int no, i, prov = 0;
  printf("Write a number: ");
  scanf("%d", &no);

  for (i = 2; i <= no / 2; ++i) {
    if (no % i == 0) {
      prov = 1;
      break;
    }
  }

  if (no == 1) {
    printf("1 is a prime number.");
  } 
  else {
    if (prov == 0)
      printf("%d is a prime number.", no);
    else
      printf("%d is not a prime number.", no);
  }
}


int main() {

  primes();

  return 0;
}

File: Functions2/ex09.py
def prime(user):
    if (user == 1):
        return False
    elif (user == 2):
        return True;
    else:
        for i in range(2,user):
            if((user % i) == 0):
                return False
        return True 

user = int(input("Write a number and lets see if its a prime number: "))
print(prime(user))